docker-compose.yml

```diff
@@ -32,7 +32,6 @@ services:
     depends_on:
       - statsd
       - localstack-s3
-      - postgresql
```

Comment:
We no longer need the postgres container to depend on the postgresql service running in the local dev environment.